deploy/openshift/templates/che-server-template.yaml

```diff
@@ -0,0 +1,279 @@
+  roleRef:
+    name: admin
+  subjects:
+  - kind: ServiceAccount
+    name: che
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: che
+    name: che-host
+  spec:
+    ports:
+    - name: http
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      app: che
+- apiVersion: v1
+  kind: Route
+  metadata:
+    labels:
+      app: che
+    name: che
+  spec:
+    to:
+      kind: Service
+      name: che-host
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: che
+    name: che
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 2
+    selector:
+      app: che
+    strategy:
+      type: ${STRATEGY}
+    template:
+      metadata:
+        labels:
+          app: che
+      spec:
+        containers:
+        - env:
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: CHE_HOST
+            value: che-${NAMESPACE}.${ROUTING_SUFFIX}
+          - name: CHE_PORT
+            value: '8080'
+          - name: CHE_API
+            value: "${PROTOCOL}://che-${NAMESPACE}.${ROUTING_SUFFIX}/api"
+          - name: CHE_WEBSOCKET_ENDPOINT
+            value: "${WS_PROTOCOL}://che-${NAMESPACE}.${ROUTING_SUFFIX}/api/websocket"
+          - name: CHE_DEBUG_SERVER
+            value: 'false'
+          - name: CHE_INFRASTRUCTURE_ACTIVE
+            value: openshift
+          - name: CHE_INFRA_KUBERNETES_BOOTSTRAPPER_BINARY__URL
+            value: "${PROTOCOL}://che-${NAMESPACE}.${ROUTING_SUFFIX}/agent-binaries/linux_amd64/bootstrapper/bootstrapper"
+          - name: CHE_INFRA_KUBERNETES_MACHINE__START__TIMEOUT__MIN
+            value: '5'
+          - name: CHE_INFRA_KUBERNETES_MASTER__URL
+          - name: CHE_INFRA_KUBERNETES_OAUTH__TOKEN
+            value: "${TOKEN}"
+          - name: CHE_INFRA_KUBERNETES_PASSWORD
+            value: "${OPENSHIFT_PASSWORD}"
+          - name: CHE_INFRA_OPENSHIFT_PROJECT
+            value: "${CHE_INFRA_OPENSHIFT_PROJECT}"
+          - name: CHE_INFRA_KUBERNETES_PVC_STRATEGY
+            value: "${CHE_INFRA_KUBERNETES_PVC_STRATEGY}"
+          - name: CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS
+            value: 'true'
+          - name: CHE_INFRA_OPENSHIFT_TLS__ENABLED
+            value: "${TLS}"
+          - name: CHE_INFRA_KUBERNETES_TRUST__CERTS
+            value: "${TLS}"
+          - name: CHE_INFRA_KUBERNETES_USERNAME
+            value: "${OPENSHIFT_USERNAME}"
+          - name: CHE_LOCAL_CONF_DIR
+            value: "/etc/conf"
+          - name: CHE_LOGS_DIR
+            value: "/data/logs"
+          - name: CHE_LOG_LEVEL
+            value: INFO
+          - name: CHE_MULTIUSER
+            value: "${CHE_MULTIUSER}"
+          - name: CHE_KEYCLOAK_AUTH__SERVER__URL
+            value: "${CHE_KEYCLOAK_AUTH__SERVER__URL}"
+          - name: CHE_JDBC_URL
+            value: "${CHE_JDBC_URL}"
+          - name: CHE_OAUTH_GITHUB_CLIENTID
```

Comment:
How can it be configured if there is no specified value?

Reply:
Fixed